pywhat/helper.py

```diff
@@ -26,18 +31,31 @@ class InvalidTag(Exception):
     pass
 
 
-def load_regexes() -> list:
-    path = "Data/regex.json"
-    fullpath = os.path.join(os.path.dirname(os.path.abspath(__file__)), path)
+def read_json(path: str):
+    fullpath = os.path.join(os.path.dirname(os.path.abspath(__file__)), "Data/" + path)
     with open(fullpath, "rb") as myfile:
-        regexes = json.load(myfile)
+        return json.loads(myfile.read())
+
+
+@lru_cache()
```

Comment:
Why do we cache this? Do we call this more than once? :)

Reply:
Yes, in API mode if many distributions/identifiers are being used. +it *greately* improves the speed of tests.